src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -1616,9 +1616,12 @@ QueueToadlet.insertFileBrowseLabel=Upload a file from disk
 QueueToadlet.insertFileBrowseButton=Browse files
 QueueToadlet.insertFileResetForm=Reset Form
 QueueToadlet.key=Key
-QueueToadlet.lastActivity=Last Activity
+QueueToadlet.lastActivity=Last Progress
```

Comment:
First notice what I answered to the below question of yours:

> > The success date is listed as "progress." Why not have the variable names be lastProgress to match? (FCP too.)
>
> This is because I unfortunately realized that "progress" is better *after* I wrote the code :( 
> It would be quite a bit of work to refactor the code since "success" is used in many places, so I thought I could use the common practice of renaming stuff by l10n only in case where the code is too much.
> 
> Do you want me to fix this or keep as is?

My answer depends what your answer is there.
If  we keep using "activity" for variables in the code, while having "Progress" in the l10n string, then I would say that we should keep using "activity" for the l10n key as well: The workaround of having a mismatch between l10n value and in-code variable naming should be apparent from the l10n file, since the l10n file is the place where it was started. Mismatching the key name and value there is the only way to indicate this mismatch, since comments are not possible in l10n files (or are they?)
Do you agree with that?